docs/_quarto.yml

```diff
@@ -100,7 +100,8 @@ website:
       contents:
         - install.qmd
         - auto: tutorials/*.qmd
-        - auto: tutorials/data-platforms
+        - auto: tutorials/cloud-data-platforms
+        - auto: tutorials/open-source-frameworks
```

Comment:
apologies to request another change here -- can you switch this to "open-source-software" and add the redirect for the data platforms?